src/app/mat-select-search/mat-select-search.component.ts

```diff
@@ -285,6 +285,30 @@ export class MatSelectSearchComponent implements OnInit, OnDestroy, AfterViewIni
             this.changeDetectorRef.markForCheck();
           });
       });
+
+      if (this.matSelect._keyManager) {
```

Comment:
could you move your implementation into a separate function? just to make it easier to read...

Reply:
Done, sorry about that!